scripts/update_version.py

```
@@ -133,435 +142,478 @@ def _extract_changelog_between(self, content: str, start_marker: str, end_marker
-        
+
         # Remove trailing empty lines
         while changelog_lines and not changelog_lines[-1].strip():
             changelog_lines.pop()
-        
-        return '\n'.join(changelog_lines).strip()
-    
+
+        return "\n".join(changelog_lines).strip()
+
     def _clean_changelog_sections(self, changelog: str) -> str:
```

Ruff (0.12.2)

`_clean_changelog_sections` is too complex (11 > 10)

(C901)